docs/getting-started/cases-req.asciidoc

```diff
@@ -43,12 +53,10 @@ a|
 NOTE: Before a user can be assigned to a case, they must log into {kib} at least
 once, which creates a user profile.
 
-| Give view-only access for cases | **Read** for the *Security* feature and **All** for the *Cases* feature
-
-| Give access to view and delete cases
-a| **Read** for the *Cases* feature under *Security* with the *Delete* sub-feature selected
+| Give view-only access for cases
+a| **Read** for the *Security* feature and **All** for the *Cases* feature
 
-NOTE: These privileges also enable you to delete comments and alerts from a case.
+NOTE: By default, `Read` for the *Cases* feature does not include authority to delete cases or delete alerts and comments from cases. You also cannot view or edit case settings. You can enable these actions by customizing the sub-feature privileges.
```

Comment:
I feel as though first sentence might a bit redundant since the note is tied to a single bullet. WDYT about removing it and making the phrasing more positive (i.e., "you can do this..." instead of "by default, you can't do this...")?

```suggestion
NOTE: Customize the sub-feature privileges to allow access to deleting cases, deleting alerts and comments from cases, and viewing or editing case settings. 
```

Reply:
Same as above, tiny tweak to Nastasha's suggestion.
```suggestion
NOTE: You can customize the sub-feature privileges to allow access to deleting cases, deleting alerts and comments from cases, and viewing or editing case settings.
```